runtime/doc/windows.txt

```diff
@@ -439,18 +439,17 @@ CTRL-W l	Move cursor to Nth window right of current one.  Uses the
 
 CTRL-W w					*CTRL-W_w* *CTRL-W_CTRL-W*
 CTRL-W CTRL-W	Without count: move cursor to the |focusable| window
-		below/right of the current one.  If there is no (focusable)
-		window below or right, go to top-left window. With count: go
-		to Nth window (windows are numbered from top-left to
-		bottom-right).  To obtain the window number see |bufwinnr()|
-		and |winnr()|.  When N is larger than the number of windows go
-		to the last window.
+		below/right of the current one.  If none, go to the top-left
+		window.  With count: go to Nth window (numbered top-left to
+		bottom-right), reversing past unfocusable windows.  To obtain
```

Comment:
Don't like the term "reversing past" here, but couldn't think of anything better that's still terse and explicit about skipping past to smaller winnrs... (or maybe it should skip forward?)

Reply:
Actually, skipping forward feels better